projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts

```diff
@@ -7,12 +7,22 @@
 import * as quote from '../../../../helpers/quote';
 
 const POWERTOOLS = 'powertools-spa';
-const testProductHammerDrillingId = '3887130';
-const testProductHammerDrillingName = 'DH40MR';
-const EMAIL = 'gi.sun@pronto-hw.com';
-const PASSWORD = '12341234';
-const USER = 'Gi Sun';
+const TESTPRODUCTHAMMERDRILLINGID = '3887130';
+const TESTPRODUCTHAMMERDRILLINGNAME = 'DH40MR';
```

Comment:
```suggestion
const TEST_PRODUCT_HAMMER_DRILLING_ID = '3887130';
const TEST_PRODUCT_HAMMER_DRILLING_NAME = 'DH40MR';
```
please use underscore to improve readability of constant names